### ISSUE TITLE
Support SDN to OVN conversion

### ISSUE THREAD

**Issue by 0kashi** [Azure/OpenShift#336]

**Comment by 0kashi** [comment]
For clusters created before 4.11, the default network was SDN.  As OpenShift SDN is deprecated as of v4.14 with removal in 4.17.  Support conversion to OVN-Kubernetes.  See the following for more details: https://access.redhat.com/articles/7083046

**Comment by jpollardRH** [comment]
Docs are now live at https://learn.microsoft.com/en-us/azure/openshift/howto-sdn-to-ovn - should this item now be marked as Done?

**Comment by 0kashi** [comment]
Yes.  https://learn.microsoft.com/en-us/azure/openshift/howto-sdn-to-ovn for documentation.
